src/app/modules/spcp/spcp.oidc.client.cache.ts

```diff
@@ -36,7 +36,7 @@ const INITIAL_REFRESH_TIMEOUT = 10000
  * Cache class which provides read-through capability and refresh-ahead before expiry
  * Handles discovery and retrieval of NDI's public jwks
  */
-export class SpOidcClientCache {
+export class SpcpOidcBaseCilentCache {
```

Comment:
Typo in class name as well
```suggestion
export class SpcpOidcBaseClientCache {
```

Reply:
fixed in f7919072bf9508d1d36d8bd709977a4d2b036d2f